src/Razor/test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/MSBuildIntegrationTestBase.cs

```diff
@@ -42,6 +47,10 @@ internal static ProjectDirectory Project
             set { _project.Value = value; }
         }
 
+        // Wether to use a local cache or not to prevent polluting the global cache
+        // with test packages.
+        public bool UseLocalPackageCache { get; set; }
```

Comment:
Is this just for debugging?

Reply:
No, it’s to not pollute CI machines with test packages. Nate recommended I did it this way.